src/api/secrets.ts

```diff
@@ -0,0 +1,47 @@
+import { ipcMain } from 'electron';
+import keytar from 'keytar';
+
+import { IPC_CHANNELS, KEYTAR_SERVICE } from '@config';
+import { SecretsRequest, SecretsRequestType, SecretsResponse, TUuid } from '@types';
+import { decrypt, encrypt, hashPassword } from '@utils/encryption';
+
+// @todo STORES HASHED PASSWORD FOR ENCRYPTION - THINK ABOUT THIS
+let encryptionKey: string;
+
+const SetEncryptionKey = (key: string) => (encryptionKey = hashPassword(key));
```

Comment:
Function names should start with a lowercase letter (`camelCase`). Here and for the other functions as well.